packages/cypress/integration/Vapor/ActionableItem.spec.js

```diff
@@ -0,0 +1,43 @@
+/// <reference types="cypress" />
+
+context('Actions', () => {
+  before(() => {
+    cy.visit('http://192.168.0.143:8080/#/components/ActionableItem')
+  })
+  it('Validate Actionnable item without onItemClick Method',() => {
+    cy.get('#actionable-item-1 div[class*="actionable-item-content"]')
+      .should('not.have.class', 'cursor-pointer')
+  })
+  it('Validate dot menu on Actionnable item without onItemClick', () => {
+    const stub = cy.stub();
+    cy.on('window:alert', stub)
+    cy.get('#actionable-item-1 div[class*="actionable-item-dots"]')
```

Comment:
Getting elements of the page using such precise selectors will make the tests very prone to breaking. I think using the testing library I linked above will make these much more robust

Reply:
Will be looking at the testing librarie thanks for the head's up!